src/components/Foremost/Foremost.tsx

```diff
@@ -172,18 +174,20 @@ const ForemostTool = () => {
                     placeholder={"Specify types (comma-separated) e.g., jpg,doc. if blank will retrieve all."}
                     {...form.getInputProps("types")}
                 />
-                <Stack spacing="lg">
-                    {/* Quiet Mode */}
-                    <Checkbox
-                        label={"Quiet Mode - enables quiet mode. Suppress output messages."}
-                        {...form.getInputProps("quiet" as keyof FormValuesType)}
+                {!checkedVerbose && (
```

Comment:
Consider using a single switch statement. This reduces code complexity.
I have tested the below for you as working, however, it requires you first rectify camelCase naming.

```
<Switch
    label={checkedQuiet ? "Verbose Mode" : "Quiet Mode"}
    checked={checkedQuiet ? checkedVerbose : !checkedVerbose}
    onChange={(e) => {
        if (checkedQuiet) {
            setCheckedVerbose(e.currentTarget.checked);
        } else {
            setCheckedQuiet(e.currentTarget.checked);
        }
    }}
/>
```

- [ ] Review code reccomendation.
- [ ] Implement as required.

Reply:
The code is not suitable for the issue, it has a default mode also, without quiet and verbose flags. Single switch statements cannot handle three scenarios